python/src/spark_rapids_ml/feature.py

```diff
@@ -192,11 +192,14 @@ def _cuml_fit(
             pdesc = PartitionDescriptor.build(
                 params[param_alias.part_sizes], params[param_alias.num_cols]
             )
+            data_arrays = [x for x, _, _ in dfs]
+            # reverse list order to compensate for cuda managed memory LRU eviction
```

Comment:
Does reversed list reduce runtime?

Reply:
Yes in the border line oversubscribed case.   See the 24 GB row for PCA with LRU Opt vs the non-LRU Opt, cuml fit column.